index.html

```diff
@@ -1,110 +1,16 @@
 ---
 layout: default
 ---   
-<div class="greenhead">Where can I see the RFP?</div>
+
+{% for faq in site.data.faq %}
```

Comment:
At this point I need to add updates and want to avoid merge conflicts. We're moving in the right direction.